allensdk/core/lazy_property/lazy_property.py

```diff
@@ -19,4 +20,7 @@ def __set__(self, obj, value):
         raise AttributeError("Can't set LazyLoadable attribute")
 
     def calculate(self):
-        return self.api_method(*self.args, **self.kwargs)
\ No newline at end of file
+        result = self.api_method(*self.args, **self.kwargs)
+        for wrapper in self.wrappers:
```

Comment:
is there a need to have a chain here?

Reply:
not a functional need of course, I figured that:
```Python
self.value = LazyProperty(self.api.get_value, wrappers=[change_one_thing, change_another_thing])
```
was more explicit than:
```Python
self.value = LazyProperty(self.api.get_value, wrappers=[change_some_things])
```

Of course, the way I've actually used the wrappers is more like the latter version, but I would like to leave the former as an option.